src/software/gui/generic_widgets/play_info/play_info_widget.cpp

```diff
@@ -4,8 +4,8 @@ PlayInfoWidget::PlayInfoWidget(QWidget* parent) : QTextEdit(parent) {}
 
 void PlayInfoWidget::updatePlayInfo(const PlayInfo& play_info)
 {
-    QString refbox_game_state_string =
-        QString("Refbox Game State: %1\n")
+    QString play_type_string =
+        QString("Refbox Gamestate: %1\n")
```

Comment:
looks like a bad merge: what does play type mean?

Reply:
Whoops changed the text but not the variable. Fixed.